tests/test_database.py

```diff
@@ -136,11 +136,60 @@ def test_map_files(num_workers):
 )
 def test_save_and_load(tmpdir, db, storage_format, num_workers):
 
+    db.meta = {}
     db.save(
         tmpdir,
         storage_format=storage_format,
         num_workers=num_workers,
     )
+    expected_formats = [storage_format]
+    for table_id in db.tables:
+        for ext in audformat.define.TableStorageFormat.attribute_values():
+            table_file = os.path.join(tmpdir, f'db.{table_id}.{ext}')
+            if ext in expected_formats:
+                assert os.path.exists(table_file)
+            else:
+                assert not os.path.exists(table_file)
+
+    # Test update other formats
+    if (
+            storage_format == audformat.define.TableStorageFormat.CSV
+            and db.tables
+    ):
+        db2 = audformat.testing.create_db()
+        db2.meta = {}
```

Comment:
Why do we need this?

Reply:
I removed them, we don't need them.

They are leftovers from me trying to get the test working for `create_db(minimal=True)` as well. But in the end I decided to just skip the test then as we don't have tables there anyway.